src/main/scala/com/bazaarvoice/sswf/model/history/HistoryFactory.scala

```diff
@@ -163,7 +178,7 @@ object HistoryFactory {
             startedTimers.put(timerId, h)
 
             assert(scheduledStep == currentStep, s"id[$scheduledStep] != currentStep[$currentStep]")
-            Some(StepEvent[StepEnum](eventId, eventId, Left(scheduledStep), "STARTED", currentStepStart, None, Duration.ZERO))
+            Some(StepEvent[StepEnum](eventId, eventId, Left(scheduledStep), "STARTED", currentStepStart, None, Duration.ZERO, invocations))
```

Comment:
Yes, thanks for pointing this out. I will reset invocations if activity transition happens.